pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -8,11 +8,12 @@ function omf.repo.pull
   set -l repo_dir $argv[1]
 
   function omf.repo.git -V repo_dir
-    command git --git-dir "$repo_dir"/.git --work-tree "$repo_dir" $argv
+    command git -C "$repo_dir" $argv
   end
 
-  function omf.repo.git.stash -V repo_dir
-    command git -C "$repo_dir" stash $argv
+  function omf.repo.after_clean -a return_code
```

Comment:
Although the cleanup function which return the parameter passed and cleans up functions is a nice idea, I found it somewhat cumbersome. But on the other side, the previous implementation had a bug, which removed the inner functions only when succeeded. I don't know a better way to solve this issue, but I would avoid masking the return code.